addons/im_livechat/static/tests/embed/feedback_panel.test.js

```diff
@@ -96,8 +100,12 @@ test("Closing folded chat window should open it with feedback", async () => {
     await contains(".o-mail-Message-content", { text: "Hello World!" });
     await click("[title='Fold']");
     await waitNotifications([env, "discuss.Thread/fold_state"]);
-    await contains(".o-mail-ChatBubble");
-    await click("[title*='Close Chat Window']");
-    await contains(".o-mail-ChatBubble", { count: 0 });
+    await click(".o-mail-ChatHub-bubbleBtn");
+    await click("[title='Close Chat Window (ESC)']");
```

Comment:
Keep the `title*='Close Chat Window'`, the `(ESC)` feels not that relevant and more a soft-spec. In the sense that we could probably change it frequently/often, and it sucks to adapt so many tests just because of that.